Dialog docs (#2409)
microsoft/teams-ai: book/src/5.in-depth-guides/2.dialogs/README.md

# Dialogs

> Dialogs are a helpful paradigm in Teams which can help improve interactions between your agent and users. When dialogs are **invoked**, they pop open a window for a user in the Teams client. The content of the dialog can be supplied by the agent application.

Reviewer: ```suggestion
Dialogs are a helpful paradigm in Teams which improve interactions between your agent and users. When dialogs are **invoked**, they pop open a window for a user in the Teams client. The content of the dialog can be supplied by the agent application.
```

## Key benefits

> 1. Dialogs pop open for a user in the Teams client. This means in group-settings, the activity that the user performs in the dialog is not visible to other users in the channel. This can keep conversations clutter-free.

Reviewer: ```suggestion
1. Dialogs pop open for a user in the Teams client. This means in group-settings, dialog actions are not visible to other users in the channel, reducing clutter.
```
